resources/timerbar.ts

```diff
@@ -419,10 +416,13 @@ export default class TimerBar extends HTMLElement {
 
     // To start full and animate to empty, we animate backwards and flip
     // the direction.
-    if (this._towardRight !== this._fill)
-      this.foregroundElement.style.transformOrigin = '100% 0%';
-    else
-      this.foregroundElement.style.transformOrigin = '0% 0%';
+    if (this._towardRight) {
```

Comment:
```suggestion
    if (this._towardRight !== this._fill) {
```

I think if it says "fill towards left" that it should be filling from right to left.

If you checkout the revision before cc7c6762, that's what it used to do:

![image](https://user-images.githubusercontent.com/462317/115125537-a1403480-9f7d-11eb-9e28-906fa092e786.png)

It makes sense to me that the empty and fill towards left would be aligned but have their colors reversed.